apps/remix-ide/src/app/compiler/compiler-helpers.js

```diff
@@ -4,17 +4,19 @@ import { Compiler } from '@remix-project/remix-solidity'
 import CompilerAbstract from './compiler-abstract'
 
 export const compile = async (compilationTargets, settings) => {
-  return await (() => {
+  let res = await (() => {
```

Comment:
`res` can be `const`

Reply:
i think this was causing issues this is why i change it to let. will try const again